heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java

```diff
@@ -1,264 +0,0 @@
-// Copyright 2018 Twitter. All rights reserved.
```

Comment:
why this file is here?

Reply:
it came in from merge conflict.

CONFLICT (rename/delete): heron/healthmgr/src/java/com/twitter/heron/healthmgr/HealthManagerMetrics.java deleted in f444372de2749c8a6953ff98a5178550ec026cd8 and renamed to heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java in HEAD. Version HEAD of heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java left in tree.


Deleted merge conflict for 'heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManagerMetrics.java':
  {local}: created file
  {remote}: deleted
Use (c)reated or (d)eleted file, or (a)bort? d